src/components/Drawer/Drawer.scss

```diff
@@ -280,7 +280,7 @@ body.overlay {
 }
 
 body.container {
-  overflow: auto !important;
+  overflow: visible !important;
```

Comment:
@Farhan-emgage has this been tested to confirm it does not create an issue with wide content displaying outside the drawer?

Reply:
@weegrom yes i tested it. when the drawer is open at that time bodu.container class is apply on body and when drawer is close then this class is removed